### DISCUSSION TITLE
Mi FOV anda loco

### DISCUSSION THREAD

**Discussion by QuicoHernandez** [Stellarium/stellarium#3699] [Q&A] [answered]
Hola, cuando coloco el FOV  en 0º la rotación de la cámara  no coincide con la orientación del cielo, para que coincida en 0º tengo que rotar 54º +-

**Comment by 10110111** [comment]
What do you mean by setting FoV to 0°? It shouldn't even be possible. Please give the exact sequence of your actions.

**Reply by QuicoHernandez** [reply, marked as answer]
Hola, solucionado. Herramientas - configurar Telescopio - Activar montura ecuatorial. Ahora todo perfecto. Saludos

![Solución](https://github.com/Stellarium/stellarium/assets/164483839/b10e7936-c4c0-4cef-ad3b-05906aed3dcd)
